devops/README.md

```diff
@@ -0,0 +1,71 @@
+# OSConfig DevOps
+This folder contains various folders in use by OSConfig's Engineering Systems (Build, Packaging, Test Infrastructure). This document serves to describe the various components involved.
+
+ * debian - Debian packaging maintainer Scripts
+ * docker - Container definitions
+ * pipeline - Azure Pipeline pipeline definitions
+ * scripts - various scripts for installing dependencies and creating service principals for the infrastructure
+ * terraform - Terraform scripts used for creating end-to-end test infrastructure
+
+# Containerized Environments
+OSConfig's engineering systems make use of containers for building, unit testing, and packaging. The container environments are located under the [docker](https://github.com/Azure/azure-osconfig/tree/main/devops/docker) directory naming convention is as follows `[distro]-[architecture]`. The container environments contain everything needed to build/test/package OSConfig.
+
+### Supported Environments
+* Debian 9
+* Ubuntu 18.04/20.04 (LTS)
```

Comment:
Suggested to list them all 3 separately

Debian 9
Ubuntu 18.04
Ubuntu 20.04

+ "LTS" is not needed, that is implied by version numbers.